consensus/view_change.go

```diff
@@ -150,10 +152,18 @@ func (consensus *Consensus) onViewChange(msg *msg_pb.Message) {
 		return
 	}
 	newLeaderKey := recvMsg.LeaderPubkey
-	if !consensus.PubKey.IsEqual(newLeaderKey) {
+	if !consensus.PubKey.Contains(newLeaderKey) {
```

Comment:
change PubKey to PubKeys, similar to consensus.PriKey to consensusPriKeys

Reply:
Okay. I have made the change